README.md

```diff
@@ -183,25 +210,23 @@ We'll use deluge Docker image from linuxserver, which runs both the deluge daemo
 ```yaml
 version: '3'
 services:
-
   deluge:
     container_name: deluge
-    image: linuxserver/deluge:102
-    network_mode: host # run on the host network directly, web UI on port 8112
+    image: linuxserver/deluge:latest
+    restart: always
+    network_mode: service:vpn # run on the vpn network
```

Comment:
I think the initial idea here was to show how to setup deluge without the VPN first, since it's not an absolute requirement to use a VPN for this setup.
Then later on (in the VPN section) introduce how to make it use the vpn network.
What do you think?

Reply:
I think keeping the vpn in the default configuration is more responsible, as not having it might make people not think it is important, which is very wrong for most people, especially in the US.

However, yeah it is probably a good idea to add a section about specifically removing the VPN configuration from the default docker-compose (removing the section, then change the deluge network_mode).